framework/looper.py

```diff
@@ -167,14 +168,14 @@ def doSigUsr2(sig,frame):
         self.setup = Setup(config, services)
 
         ######### Save processing information to output directory
-
-        # save the versions
-        if self.config.versions:
+        
+        if hasattr(self.config, 'versions'):
             self.config.versions.write_yaml('/'.join([self.outDir,
                                                       'software.yaml']))        
         # remove versions from the config as it can't be pickled
-        config_no_versions = copy.copy(self.config)
-        delattr(config_no_versions, 'versions')
+        #config_no_versions = copy.copy(self.config)
```

Comment:
I don't see how this version stuff could be related to these condor developments -> should stay.
What is the problem with that?

Reply:
need to clarify